.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
**[INVALID]**

- Hey, you've done well adding the linter file, but while you did add the linters, you actually added the wrong linter and you've edited the linter file which is against the code of conduct in microverse regarding the linters. Please follow my instructions closely to update the linters in your project.

1. Please, kindly ***delete*** the `.eslintrc.json` file from your project as this is not needed for this milestone.
2. Please, kindly ***delete*** the contents in `.github/workflows/linters.yml` and ***replace*** them with the content in this [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/337cdadeb1abe79adf6f02b6361ef6db09b0244e/html-css/.github/workflows/linters.yml#L1-L56)
3. Please, kindly ***delete*** the contents in `.stylelintrc.json` and ***replace*** them with the contents in this [.stylelintrc.json file](https://github.com/microverseinc/linters-config/blob/337cdadeb1abe79adf6f02b6361ef6db09b0244e/html-css/.stylelintrc.json#L1-L22)
4. After, steps 1 - 3, delete your `node_modules` directory in your project.
5. Run `npm install` in your terminal to install the dependencies. OR
6. Implicitly install the linter using npm install below
- stylelint: `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
`
- webhint: `npm install --save-dev hint@7.x`
7. Fix your linter error, if you have any.